RELEASE_NOTES.md

```diff
@@ -1,5 +1,7 @@
 Release Notes
 =============
+## 1.6.22
```

Comment:
iirc, @TheRSP told me to set this to vNext and the Farmer maintainers will update this when they release (Might be multiple things in a release)

Reply:
Wait, that probably doesn't apply in our fork?

Reply:
yes, I am not sure if they include it in 1.6.22 release or not. Updated to vNext